client/state/themes/selectors/theme-preview-visibility.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import 'state/themes/init';
+
+/**
+ * Returns the ThemePreview state
+ *
+ * @param  {object}  state Global state tree
+ * @returns {?string}  ThemePreview state
+ */
+export function themePreviewVisibility( state ) {
+	return state.themes.themePreviewVisibility ?? null;
```

Comment:
I think that the `??` null` shouldn't be necessary, as the default state for that reducer subtree is `null` anyway.

Reply:
Good catch!